<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GamePanel</class>
 <widget class="QMainWindow" name="GamePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>1200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>1239</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1600</width>
     <height>1200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1600</width>
     <height>1200</height>
    </size>
   </property>
   <widget class="QLabel" name="Gamepanel_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>1600</width>
      <height>1200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1600</width>
      <height>1200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1600</width>
      <height>1200</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>game_resources/football_Field_image2(2).jpg</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="Score_Timer_Panel" native="true">
    <property name="geometry">
     <rect>
      <x>1300</x>
      <y>0</y>
      <width>300</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>80</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="Myteam_label">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>我的队伍</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="3">
      <widget class="QLabel" name="Timer_label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="Myheadimage_label">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>game_resources/headimage_test2(1).png</pixmap>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="Robotscore_lineedit">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="Robot_label">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Robot</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="Ratio_label">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="Myscore_lineedit">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="Robotheadimage_label">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>game_resources/headimage_test1(1).png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>850</y>
      <width>86</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </widget>
   <widget class="QWidget" name="MyPlayer5" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>550</y>
      <width>86</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <widget class="QLabel" name="My_goal_keeper5">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="MyP5">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>game_resources/gameplayer_image12_Matinez(1).png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="MyPlayer3" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>340</y>
      <width>86</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="MyP3">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>game_resources/gameplayer_image3_Suarez(1).png</pixmap>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="My_goal_keeper3">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="MyPlayer1" native="true">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>430</y>
      <width>86</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="MyP1">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>game_resources/gameplayer_image2_Messi(1).png</pixmap>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="My_goal_keeper1">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources_file.qrc">:/game_resources/football_image2(1).jpg</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="MyPlayer2" native="true">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>600</y>
      <width>86</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="1">
      <widget class="QLabel" name="My_goal_keeper2">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="MyP2">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>game_resources/gameplayer_image6_Neymar(1).png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="MyPlayer4" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>710</y>
      <width>86</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_7">
     <item row="0" column="0">
      <widget class="QLabel" name="MyP4">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>game_resources/gameplayer_image4_Maradona(1).png</pixmap>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="My_goal_keeper4">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="RobotPlayer1" native="true">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>430</y>
      <width>86</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_8">
     <item row="0" column="0">
      <widget class="QLabel" name="Robot_goal_keeper1">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="Robot_P1">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>game_resources/gameplayer_image1_Cristiano(1).png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="RobotPlayer2" native="true">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>600</y>
      <width>86</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_9">
     <item row="0" column="0">
      <widget class="QLabel" name="Robot_goal_keeper2">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="Robot_P2">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>game_resources/gameplayer_image7_Mbappe(1).png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="RobotPlayer3" native="true">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>340</y>
      <width>86</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_10">
     <item row="0" column="0">
      <widget class="QLabel" name="Robot_goal_keeper3">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="Robot_P3">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>game_resources/gameplayer_image10_LukaModvic(1).png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>760</y>
      <width>86</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_11"/>
   </widget>
   <widget class="QWidget" name="RobotPlayer4" native="true">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>710</y>
      <width>86</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_12">
     <item row="0" column="0">
      <widget class="QLabel" name="Robot_goal_keeper4">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="Robot_P4">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>game_resources/gameplayer_image5_Flix(1).png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="RobotPlayer5" native="true">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>550</y>
      <width>86</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_13">
     <item row="0" column="0">
      <widget class="QLabel" name="Robot_goal_keeper5">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="Robot_P5">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>game_resources/gameplayer_image14_Courtois(1).png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Robot_goal">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>490</y>
      <width>51</width>
      <height>181</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>181</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>181</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources_file.qrc">:/game_resources/goal(2).png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="My_goal">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>51</width>
      <height>181</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>51</width>
      <height>181</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>51</width>
      <height>181</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources_file.qrc">:/game_resources/goal(1).png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="Settings_menu">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionvoiceforce"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionbackground"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="Helper_menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="helper_action"/>
   </widget>
   <widget class="QMenu" name="MoreOperations_menu">
    <property name="title">
     <string>更多操作</string>
    </property>
    <addaction name="stop_action"/>
    <addaction name="separator"/>
    <addaction name="continue_action"/>
    <addaction name="separator"/>
    <addaction name="restart_action"/>
    <addaction name="replace_action"/>
   </widget>
   <addaction name="Settings_menu"/>
   <addaction name="Helper_menu"/>
   <addaction name="MoreOperations_menu"/>
  </widget>
  <action name="actionvoiceforce">
   <property name="text">
    <string>voiceforce_settings</string>
   </property>
  </action>
  <action name="actionbuttonsettings">
   <property name="text">
    <string>buttonsettings</string>
   </property>
  </action>
  <action name="actionbackground">
   <property name="text">
    <string>background_settings</string>
   </property>
  </action>
  <action name="actionrole">
   <property name="text">
    <string>role/team</string>
   </property>
  </action>
  <action name="actionemit">
   <property name="text">
    <string>emit</string>
   </property>
  </action>
  <action name="actionheadimage">
   <property name="text">
    <string>headimage</string>
   </property>
  </action>
  <action name="stop_action">
   <property name="text">
    <string>stop</string>
   </property>
  </action>
  <action name="actionrestart">
   <property name="text">
    <string>restart</string>
   </property>
  </action>
  <action name="actionreplace_the_soccer_player">
   <property name="text">
    <string>replace the soccer player</string>
   </property>
  </action>
  <action name="continue_action">
   <property name="text">
    <string>continue</string>
   </property>
  </action>
  <action name="restart_action">
   <property name="text">
    <string>restart</string>
   </property>
  </action>
  <action name="helper_action">
   <property name="text">
    <string>gamecontrol/introduction</string>
   </property>
  </action>
  <action name="actionintroduction">
   <property name="text">
    <string>introduction</string>
   </property>
  </action>
  <action name="actionrole_settings">
   <property name="text">
    <string>role_settings</string>
   </property>
  </action>
  <action name="replace_action">
   <property name="text">
    <string>replace the soccer player</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources_file.qrc"/>
 </resources>
 <connections/>
</ui>
